### DISCUSSION TITLE
🧩 FEATURE REQUEST: Improved tab naviation

### DISCUSSION THREAD

**Discussion by AnishDe12020** [dailydotdev/daily#511] [Feature Requests]
## Feature Request
Improved tab navigation, a skip to the content button that skips the sidebar navigation.

### Description
It would be awesome if tab navigation could be improved the daily.dev chrome extension (and probably even the web app). There seems to be no Skip to content/Skip navigation button that pops up during tab navigation. It would be cool to have this as, otherwise, one has to tab through the whole sidebar (refer to attached screen recording).

https://user-images.githubusercontent.com/63192115/160091311-6d289a36-c7a8-4382-9a15-67ee4ee5da86.mp4

### Solution
A Skip to content/Skip navigation button (examples below)

Examples (open the link and press tab) - 
Google Search - https://www.google.com/search?q=daily.dev
My Portfolio Site - https://anishde.dev/

PS: Also looking forward to more keyboard shortcuts (like `c` to comment, `u` to upvote or things like that)

**Comment by rebelchris** [comment]
That is a great suggestion @AnishDe12020 I'll convert this to a discussion and notify the team about this 🙌
